frameworks/digital-outcomes-and-specialists-2/search_mappings/briefs.json

```diff
@@ -0,0 +1,152 @@
+{
+  "settings": {
+    "analysis": {
+      "char_filter": {
+        "nonalpha_removal": {
+          "type": "pattern_replace",
+          "pattern": "\\W+",
+          "replacement": ""
+        }
+      },
+      "normalizer": {
```

Comment:
So a normalizer is different to an analyzer, in that it guarantees that it produces a single token. But why is that important for us? Why can't we just use an analyzer?

Reply:
analyzers don't work with `keyword` field types